packages/extensions/venia-sample-language-packs/i18n/fr_FR.json

```diff
@@ -0,0 +1,366 @@
+    "orderProgressBar.deliveredText": "Livré",
+    "orderProgressBar.processingText": "En traitement",
+    "orderProgressBar.readyToShipText": "Prêt pour l'expédition",
+    "orderProgressBar.shippedText": "Expédié",
+    "orderRow.deliveredText": "Livré",
+    "orderRow.orderDateText": "Date de commande",
+    "orderRow.orderNumberText": "Ordre #",
+    "orderRow.orderTotalText": "Total de la commande",
+    "orderRow.processingText": "En traitement",
+    "orderRow.readyToShipText": "Prêt pour l'expédition",
+    "orderRow.shippedText": "Expédié",
+    "Our Story": "Notre histoire",
+    "pagination.firstPage": "passer à la première page",
+    "pagination.prevPage": "passer à la page précédente",
+    "pagination.nextPage": "passer à la page suivante",
+    "pagination.lastPage": "passer à la dernière page",
+    "priceAdjustments.couponCode": "Entrer le code promotionnel",
+    "priceAdjustments.giftOptions": "Voir les options de cadeaux",
+    "priceAdjustments.shippingMethod": "Estimez votre expédition",
+    "priceSummary.checkoutButton": "Passer à la caisse",
+    "priceSummary.errorText": "Un problème est survenu. Veuillez actualiser et réessayer.",
+    "priceSummary.estimatedTotal": "Total estimé",
+    "priceSummary.lineItemLabel": "Sous-total",
+    "priceSummary.total": "Totale",
+    "product.editItem": "Modifier l'article",
+    "product.errorFetch": "Erreur de récupération des données",
+    "product.moveToFavorites": "Déplacer vers les favoris",
+    "product.outOfStock": "En-rupture-de-stock",
+    "product.outOfStockTryAgain": "Ce produit est actuellement en rupture de stock. Veuillez réessayer plus tard.",
+    "product.price": " ch.",
+    "product.removeFromCart": "Retirer du panier",
+    "product.removeFromFavorites": "Retirer des favoris",
+    "productDetail.inStock": "En stock",
+    "productDetail.outOfStock": "En rupture de stock",
+    "productDetail.skuNumber": "FR-SKU # {sku}",
+    "productDetail.unknown": "Inconnue",
+    "productForm.dataError": "Une erreur s'est produite. Veuillez actualiser et réessayer.",
+    "productForm.fetchingProductOptions": "Récupération des options de produit ...",
+    "productForm.quantity": "Quantité",
+    "productForm.submit": "Mise à jour",
+    "productForm.updatingCart": "Mise à jour du panier ...",
+    "productFullDetail.cartAction": "Ajouter au chariot",
+    "productFullDetail.errorCart": "Un problème est survenu avec votre panier. Veuillez actualiser la page et réessayer d'ajouter l'élément.",
+    "productFullDetail.errorToken": "Un problème est survenu avec votre panier. Veuillez vous reconnecter et réessayer d'ajouter l'élément.",
+    "productFullDetail.errorUnknown": "Impossible d'ajouter l'article au panier. Veuillez vérifier les options requises et réessayer.",
+    "productFullDetail.productDescription": "Description du produit",
+    "productList.each": " ch.",
+    "productList.outOfStock": "en-rupture-de-stock",
```

Comment:
**"en rupture de stock"** does not need dash between words in french.
This seems like a direct translation of "out-of-stock".